app/src/componentsV2/CodeEditor/components/EditorV2/components/VariablePopOver/index.tsx

```diff
@@ -168,16 +211,26 @@ const VariableInfo: React.FC<{
   return (
     <>
       <div className="variable-info-property-container">
-        <Tag icon={<PiHighlighterBold />} className="variable-info-header">
-          {source.scope}
-        </Tag>
-
         {source.scope !== VariableScope.RUNTIME && (
           <>
-            <span className="variable-header-info-seperator">/</span>
-            <div className="variable-info-header-name">{source.name}</div>
+            <span>{getScopeIcon(source.scope)} </span>
+            <span className="variable-header-info-seperator"> </span>
+            <div className="variable-info-header-name"> {source.name}</div>
           </>
         )}
+
+        {/* Edit button - only for non-runtime variables */}
+        {source.scope !== VariableScope.RUNTIME && onEditClick && (
```

Comment:
Style issue for runtime variable info

<img width="316" height="217" alt="Image" src="https://github.com/user-attachments/assets/6a651205-8c92-4136-9765-5624ae777254" />